### ISSUE TITLE
Lesson 1 no output

### ISSUE THREAD

**Issue by Shadowblitz16** [cfenollosa/os-tutorial#214]
on lesson 1 there is no output from 
```
nasm -f bin boot_sect_simple.asm -o boot_sect_simple.bin
```
![image](https://user-images.githubusercontent.com/2453707/122326182-b9322700-cee0-11eb-80be-1a596fe6a897.png)
![image](https://user-images.githubusercontent.com/2453707/122326253-d8c94f80-cee0-11eb-82cb-c82af4dc77ee.png)

**Comment by Shadowblitz16** [comment]
looks like I misunderstood some of this.
nasm compiles boot.asm to boot.bin

I am still not getting a qemu window out `qemu-system-x86_64 boot.bin`

**Comment by MrHallway0** [comment]
@Shadowblitz16 could you please give a screenshot of the QEMU window?

**Comment by Shadowblitz16** [comment]
@MrHallway0 
That's the thing I couldn't get the window to appear.

With some tinkering and uninstalling and reinstalling I got a qemu window to appear but it looks like this.
![image](https://user-images.githubusercontent.com/2453707/122452811-80d12e00-cf5e-11eb-8f9d-2df3a572a68b.png)

note I had to uninstall the brew version of qemu and install the one from the apt version.

however I have to run this command.
`qemu-system-x86_64 -S -drive format=raw,file=boot.bin`
instead of..
`qemu boot.bin`
